packages/radix-ui-themes/src/components/dialog.css

```diff
@@ -1,46 +1,43 @@
 .rt-DialogOverlay {
-  display: flex;
-  align-items: center;
-  justify-content: center;
   position: fixed;
   inset: 0;
 
-  --dialog-overlay-padding-top: var(--space-4);
-  --dialog-overlay-padding-bottom: max(var(--space-4), 4vh);
-  padding-top: var(--dialog-overlay-padding-top);
-  padding-bottom: var(--dialog-overlay-padding-bottom);
-  padding-left: var(--space-4);
-  padding-right: var(--space-4);
-
-  &::after {
+  &::before {
+    position: fixed;
```

Comment:
Tried to avoid having a pseudo and use background-color on `.rt-DialogOverlay` directly and animate that, but there's some bug in Chrome when the animation to open the dialog is butchered on the initial page load (and works fine when you open the dialog again)